vyper/ast.py

```diff
@@ -43,9 +43,9 @@ def __init__(self, **kwargs):
             if field_name in self.get_slots():
                 setattr(self, field_name, value)
             elif value:
-                raise CompilerPanic(
-                    f'Unsupported non-empty value field_name: {field_name}, '
-                    f' class: {type(self)} value: {value}'
+                raise SyntaxException(
+                    f'Unsupported non-empty value (valid in Python, but invalid in Vyper) \n'
```

Comment:
I'm not convinced this is always the correct error message, but maybe it's not important.

Reply:
`SyntaxException` is probably a little clearer. There error message is fine.